tst/testinstall/kernel/stats.tst

```diff
@@ -12,6 +12,8 @@ gap> f('x');
 Error, Range: <last> must be a small integer (not a character)
 
 # test Assert with two arguments
+gap> function() Assert(fail, 0); end();
+Error, Assert: <lev> must be a small integer (not the value 'fail')
```

Comment:
For the record, I am using `<lev>` here as that matches what the documentation for `Assert` calls that argument (that's my general rule for the name of the arguments in error messages). I am fine with changing it here, e.g. to `level`, but IMHO it then should be changed in the documentation as well.